support-frontend/assets/helpers/productPrice/productOptions.ts

```diff
@@ -40,13 +40,7 @@ export type PaperAndDigitalProductOptions =
 	| typeof SixdayPlus
 	| typeof EverydayPlus;
 
-const ActivePaperProductTypes: PaperProductOptions[] = [
-	Everyday,
-	Sixday,
-	Weekend,
-	Saturday,
-	Sunday,
-];
+const ActivePaperProductTypes: PaperProductOptions[] = [Everyday, Weekend];
```

Comment:
Do we want to actively remove `Sixday`/`Saturday`/`Sunday` throughout this file and beyond? Or wait for the generic checkout to take over first and remove everything then?

Reply:
I don't think we should remove `Sixday/Saturday/Sunday` beyond the changes in this PR for now, there's a chance things will change so I don't think we should be prematurely removing any trace of the products for now until the plans for Newspapers over the next quarter or 2 are clearer.